server/sonar-server/src/main/java/org/sonar/server/computation/period/PeriodsHolderImpl.java

```diff
@@ -50,10 +50,10 @@
    */
   public void setPeriods(Iterable<Period> periods) {
     requireNonNull(periods, "Periods cannot be null");
-    checkArgument(Iterables.size(periods) <= 5, "There can not be more than 5 periods");
-    checkState(this.periods == null, "Periods have already been initialized");
+    checkArgument(Iterables.size(periods) <= MAX_NUMBER_OF_PERIODS, String.format("There can not be more than %d periods", MAX_NUMBER_OF_PERIODS));
+      checkState(this.periods == null, "Periods have already been initialized");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AIndentationCheck)